schemas/veres-one-validator.js

```diff
@@ -162,7 +162,38 @@ const didDocument = {
             type: 'string',
           }
         }
-      },
+      }
+    },
+    electorPool: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: [
+          'capability',
+          'id',
+          'type'
+        ],
+        properties: {
+          id: {
+            type: 'string'
+          },
+          type: {
+            oneOf: [{
```

Comment:
this should be `anyOf`.  `oneOf` should be thought of as `exactly one of these schemas`.  When `oneOf` is used, the validator must validate the input against every possible schema to determine that it matches exactly one.